Libraries/Components/ActivityIndicator/ActivityIndicator.js

```diff
@@ -29,9 +29,11 @@ type IndicatorSize = number | 'small' | 'large';
 
 type IOSProps = $ReadOnly<{|
   /**
-   * Whether the indicator should hide when not animating (true by default).
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#hideswhenstopped
+    Whether the indicator should hide when not animating.
+
+    https://reactnative.dev/docs/activityindicator.html#hideswhenstopped
```

Comment:
I still don't understand why we would need these URLs in the comments? The url will be generated from:
- Base url: `https://reactnative.dev/docs/`
- ActivityIndicator.js = `activityindicator.html`
- hideWhenStopped = `#hidewhenstopped`

Unless there's a reason for us to navigate from code to docs, we shouldn't need this

Reply:
It was already there in the comment. I have not removed it.

Reply:
> I still don't understand why we would need these URLs in the comments? The url will be generated from:
> 
> * Base url: `https://reactnative.dev/docs/`
> * ActivityIndicator.js = `activityindicator.html`
> * hideWhenStopped = `#hidewhenstopped`
> 
> Unless there's a reason for us to navigate from code to docs, we shouldn't need this

Yes, Indeed